src/main/java/uk/gov/hmcts/reform/prl/handlers/TransferToAnotherCourtEventHandler.java

```diff
@@ -44,6 +58,22 @@ public void notifyAllParties(final TransferToAnotherCourtEvent event) {
         sendEmailToRespondents(caseData, EmailTemplateNames.TRANSFER_COURT_EMAIL_NOTIFICATION_RESPONDENT);
 
         sendEmailToOtherParties(caseData, EmailTemplateNames.TRANSFER_COURT_EMAIL_NOTIFICATION_OTHER_PARTIES);
+
+        if (caseData.getCourtEmailAddress() != null) {
+            sendTransferToAnotherCourtEmail(event.getAuthorisation(),caseData);
+        }
+    }
+
+    private void sendTransferToAnotherCourtEmail(String authorization,CaseData caseData) {
+        try {
+            sendgridService.sendTransferCourtEmailWithAttachments(authorization,
+                                                                  getEmailProps(caseData.getApplicantCaseName(),
+                                                                                String.valueOf(caseData.getId()),
+                                                                                caseData.getIssueDate()),
+                                                                  caseData.getCourtEmailAddress(), getAllCaseDocuments(caseData));
+        } catch (IOException e) {
+            log.error("Failed to send Email to {}", caseData.getCourtEmailAddress());
```

Comment:
do not print email. print it like failed to send email for this case. this email address anyway we are storing in DB.